### DISCUSSION TITLE
taxonomy_terms_article.html

### DISCUSSION THREAD

**Discussion by huehnerlady** [google/docsy#593] [Q&A] [answered]
Hi,

I love the feature for the taxonomies. 
I currently try to use it for tags and authors in our team-internal blog. 

So far I got the cloud to work and also the tags and authors seem to render correctly. What I do not get to work is the overview of the articles containing the tags (like [this](https://www.docsy.dev/tags/).

in docsy example you can see [here](https://www.docsy.dev/tags/tag-1/) that there is a list of all blog articles with the tag `tag-`. For me there is a normal looking page with the tag name as a header and the default layout. on the left side there is a sidebar containing the content of the WHOLE Website (Documentation, blog, etc).

I thought maybe there is already an idea what I do wrong? Otherwise I can try trimming the website into a minimal example :) 

@narrenfrei maybe you can help as you implemented the feature? :)

**Comment by narrenfrei** [comment, marked as answer]
@huehnerlady Out of the box you should be able to click on such a tag (in your case: person), then you should get to page listing all articles with this tag and then you should find on the end of this age a like like "All tags" (in your case maybe: "All people"). Then you should get to page listing all tags.

Does it work as described?

**Reply by huehnerlady** [reply]
@narrenfrei that is the behaviour I was looking for, instead it opens a default layouted page with the name of the taxonomy as title and nothing inside. I do have a left sidebar containing all pages existing in the whole website and on the right I have the taxonomy cloud (something not shown in your example either)

**Reply by huehnerlady** [reply]
<img width="1766" alt="Screenshot 2021-06-15 at 09 04 39" src="https://user-images.githubusercontent.com/12533979/122008143-f7342b80-cdb8-11eb-8365-929e162a7335.png">

I made a quick screenshot as an example.

As I say if it doesn't help I am happy to try to adapt our page towards an example, just didn't wanna do the effort if you can understand the problem from description/screenshot 🙂

**Reply by narrenfrei** [reply]
Could I have a look into your repository or in a minimal example?
Is there maybe a page in your Docs with the same URL?

**Reply by huehnerlady** [reply]
[hugo.zip](https://github.com/google/docsy/files/6653638/hugo.zip)

I now removed all the team-specific data and tried to show the problem.

http://localhost:1313/tags/tag-1/ Here you can see the default layout with the sidebar.

I also newly notices that the tags cloud is visible everywhere in the system (eg. http://localhost:1313/aws/), I would like to keep it to just in the blog section. is that possible without copy/pasting?

**Reply by narrenfrei** [reply]
When I delete the following from your `content/de/_index.html` the pages for taxonomy get generated correctly:
```
cascade:
- type: "docs"
  _target:
    path: "/*/**"
```

**Reply by huehnerlady** [reply]
ah super cool, that fixed it 👍🏻 

For the other one I would create a seperate discussion :)

**Reply by narrenfrei** [reply]
> I also newly notices that the tags cloud is visible everywhere in the system (eg. http://localhost:1313/aws/), I would like to keep it to just in the blog section. is that possible without copy/pasting?

As default the clouds get generated for all blog and docs pages. With the cascade parameter in your `content/de/_index.html` you set all pages of your Docsy site to `type = docs` (I think.)

So now you should only see the clouds on docs or blog pages. If you only want the clouds on blog pages you would have to make a project specific template for you docs pages (at the moment).
